### ISSUE TITLE
Support 3rd party as a source

### ISSUE THREAD

**Issue by tomrav** [wix/stylable-intelligence#200]
Stylable Intelligence should offer support for all existing features when consuming a 3rd party package (from `node_modules`).

- [x] Completions
  - [x] extend
  - [x] CSS mixin
  - [x] var values
  - [x] selector pseudo state/elemets
  - [x] named imports  
  - [x] JS mixin
  - [x] formatter
- [x] Hinting
  - [x] JS mixin
  - [x] States with params
- [x] Go to definition
- [x] Color presentation (but not editable)

**Comment by hepiyellow** [comment]
Does this issue include the error I am getting when importing a stylesheet from an external package like here:
```
:import {
  -st-from: "wix-ui-core/index.st.css";
  -st-named: Tooltip;
}
```
The error:
```
[stylable] Imported file 'wix-ui-core/dist/src/baseComponents/Popover/Popover.st.css' not found
```

**Comment by tomrav** [comment]
Actually sounds like that error originates from Stylable, and the intelligence just displays it.
Are you running in a mono-repo project?

**Comment by hepiyellow** [comment]
Is there any config that has to be done after installing the stylable-intelligence extension?

**Comment by tomrav** [comment]
No, the intelligence extension should not require any configuration.
It does relate to what I'm doing now, will take a look.

**Comment by tempit** [comment]
Removed :vars, @namesapce etc. from todo list, as they have no relation to this feature.

**Comment by tomrav** [comment]
Resolved in #220
